test/modules/image-sequencer.js

```diff
@@ -9,177 +9,180 @@ var test = require('tape');
+		}
+	if (g) return b;
+	else return i;
 }
-var parent = (typeof(global)==="undefined")?window:global;
-var global1 = copy(true,parent);
+var parent = (typeof (global) === "undefined") ? window : global;
+var global1 = copy(true, parent);
 
 var sequencer = ImageSequencer({ ui: false });
 var red = "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAAQABADASIAAhEBAxEB/8QAFQABAQAAAAAAAAAAAAAAAAAAAAf/xAAUEAEAAAAAAAAAAAAAAAAAAAAA/8QAFQEBAQAAAAAAAAAAAAAAAAAABgj/xAAUEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwCdABykX//Z";
 
 test('Image Sequencer has tests', function (t) {
-  t.equal(true, true, "Initial Test");
-  t.end();
-});
-
-test('loadImages loads a DataURL image and creates a step.', function (t){
-  sequencer.loadImages('test',red);
-  t.equal(sequencer.images.test.steps.length, 1, "Initial Step Created");
-  t.equal(typeof(sequencer.images.test.steps[0].output.src), "string", "Initial output exists");
-  t.end();
-});
-
-test('modulesInfo() returns info for each module', function (t){
-  var info = sequencer.modulesInfo();
-  t.equal(Object.keys(info).length, Object.keys(sequencer.modules).length);
-  t.equal(info.hasOwnProperty(Object.keys(sequencer.modules)[0]), true);
-  t.equal(info[Object.keys(sequencer.modules)[0]].hasOwnProperty('name'), true);
-  t.equal(info[Object.keys(sequencer.modules)[0]].hasOwnProperty('inputs'), true);
-  t.end();
-});
-
-if(!sequencer.options.inBrowser)
-  test('loadImage loads an image from URL and creates a step. (NodeJS)', function (t){
-    require('dns').resolve('www.github.com', function(err) {
-      if (err) {
-        console.log("Test aborted due to no internet");
-        t.end();
-      }
-      else {
-        sequencer.loadImage('URL','https://ccpandhare.github.io/image-sequencer/examples/images/red.jpg', function(){
-          t.equal(sequencer.images.URL.steps.length, 1, "Initial Step Created");
-          t.equal(typeof(sequencer.images.URL.steps[0].output.src), "string", "Initial output exists");
-          t.end();
-        });
-      }
-    });
-  });
-
-if(!sequencer.options.inBrowser)
-  test('loadImages loads an image from PATH and creates a step. (NodeJS)', function (t){
-    sequencer.loadImages('examples/images/red.jpg');
-    t.equal(sequencer.images.image1.steps.length, 1, "Initial Step Created");
-    t.equal(typeof(sequencer.images.image1.steps[0].output.src), "string", "Initial output exists");
-    t.end();
-  });
-
-test('loadImage works too.', function (t){
-  sequencer.loadImage('test2',red);
-  t.equal(sequencer.images.test2.steps.length, 1, "Initial Step Created");
-  t.equal(typeof(sequencer.images.test2.steps[0].output.src), "string", "Initial output exists");
-  t.end();
+	t.equal(true, true, "Initial Test");
```

Comment:
Ah, here, i prefer we stick to two space indents -- not tabs -- as that's the style we've established in the rest of PL's systems -- i hope it's not too much trouble to change?

Reply:
Oh no problem @jywarren I'll change It now😄